cmd/clusterctl/cmd/delete.go

```diff
@@ -68,6 +68,18 @@ var deleteCmd = &cobra.Command{
 		# Cluster API Providers are orphaned and there might be ongoing costs incurred as a result of this.
 		clusterctl delete --all
 
+		# Deletes all the providers in a namespace
+		# Important! As a consequence of this operation, all the corresponding resources managed by
```

Comment:
This goes beyond this PR but I think we might be diluting the importance of "Important!" by saying it 6 times 😄